.github/workflows/publish.yml

```diff
@@ -85,6 +85,7 @@ jobs:
         env:
           CMAKE_BUILD_TYPE: Release # do not compile with debug symbol to reduce wheel size
```

Comment:
Why does this not satisfy the same requirement? We should decide on one way to specify this environment variable

Reply:
Actually that's a good point, I did not see #4602 and I looked at the binaries generated in the 0.4.2 release. Closing this as it's a duplicate. Thanks